app/src/main/scala/org/alephium/explorer/cache/CaffeineAsyncCache.scala

```diff
@@ -36,6 +79,10 @@ class CaffeineAsyncCache[K, V](cache: AsyncLoadingCache[K, V]) {
   def get(key: K): Future[V] =
     cache.get(key).asScala
 
+  def getAll(keys: Iterable[K])(implicit ec: ExecutionContext): Future[Seq[(K, V)]] =
+    //FIXME: `asScala.toSeq` could be achieved in single iteration using Factory or BuildFrom
```

Comment:
Nit: FIXME is supposed to be fixed in the PR or later?

I love optimizations like this comment ❤️

Reply:
Another PR. Thought using Factory or BuildFrom would lead to a discussion. Better done in another PR targeting this one change.